src/parser/expression.js

```diff
@@ -808,36 +795,57 @@ pp.parseObj = function (isPattern, refShorthandDefaultPos) {
   return this.finishNode(node, isPattern ? "ObjectPattern" : "ObjectExpression");
 };
 
-pp.parseObjPropValue = function (prop, startPos, startLoc, isGenerator, isAsync, isPattern, refShorthandDefaultPos) {
```

Comment:
`parseObjPropValue` was split into `parseObjectMethod` and `parseObjectProperty`